README.md

```diff
@@ -37,3 +37,25 @@ your prompt should look similar to this this:
 Run the tests on your machine:
 
     pytest -vv digicampipe
+
+## Pointing using lid CCD images
+
+To be able to determine the pointing using stars reflections on the lid, you need to install astrometry.net:
+
+    sudo apt-get install astrometry.net
+    
+You will need to download catalogs too: 
+
+    sudo mkdir ~/astrometry-data
+    cd ~/astrometry-data
+    wget http://broiler.astrometry.net/~dstn/4200/wget.sh
+    chmod +x wget.sh
+    ./wget.sh
+
+Grab a beer as it will take a while ... Also, indexes 4203 and lower are probably not needed.
+
+Then add the folowing line to /etc/astrmetry.cfg:
+
+    add_path ~/astrometry-data
```

Comment:
I do not know the commandline tool `add_path`. I assumed this is a part of astrometry.net, so I installed it on my machine ... no luck. I still don't have `add_path`

Reply:
that line should be inserted in the /etc/astrmetry.cfg file (the location whithin that file is probably not important).

Reply:
Ah .. I am to stupid to read your instructions ... sorry

Reply:
omg that 10GB

Reply:
If we let travis download that for every test, the astropy server load will spike :-D

Reply:
Is it possible to put astrometry in the initial image travis is using to build the software ?
On the long term, I think the solution using the web service is the one we should to go. However there is an API key which must be used, which probably means there is an usage cap.

Reply:
I think custom images are part of their enterprise features, i.e. we don't get it for free. Not sure is CTA is ready to actually pay for one of the most important practices in software development .....

Reply:
I guess then I will have to go for the client connecting to the webservice.

Reply:
I think we should assume that astrometry.net is installed locally, and it is a requirement during installation (not part of the digicam pipe distribution). The digicam pipe should test if the existence of astrometry.net is tested, but not the location of the current package. For tests, a mockup astrometry.net should be produced which accepts images and return astrometry.net output (as suggested by Dominik).

Reply:
Thanks for the input, but I already got the client for web API of astrometry.net working today. It makes things much easier as I can test if astrometry is available locally and, if not, use the web service.